index.html

```diff
@@ -30,16 +50,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
```

Comment:
To make use of parent/child relationships you can add multiple classes on each box or use an id here.